.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           key: gradle-cache-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}
 
       - run: git tag -l | xargs git tag -d && git fetch -t # ensure all tags are fetched and up-to-date
-      - run: ./gradlew --scan --profile --parallel --stacktrace classes testClasses
+      - run: ./gradlew --scan --profile --parallel --stacktrace -PenableErrorProne=true classes testClasses compileJava compileTestJava
```

Comment:
This looks fine, though the main question I'd have here is: do we know why this was originally separate? Want to make sure there isn't a risk of something else we might not be considering